lib/matplotlib/_mathtext_data.py

```diff
@@ -8,250 +8,238 @@
 
 from matplotlib.externals import six
 
-"""
-from matplotlib.ft2font import FT2Font
-font = FT2Font('/usr/local/share/matplotlib/cmr10.ttf')
-items = font.get_charmap().items()
-items.sort()
-
-for charcode, glyphind in items:
-    print charcode, glyphind
-"""
-
 latex_to_bakoma = {
-    r'\oint'                     : ('cmex10',  45),
```

Comment:
No, actually.

Here's the gist of this change.  TrueType fonts have the concept of `charcodes` (which loosely corresponds to a Unicode codepoint if it's a Unicode font which most are these days) and `glyph indices` (which is just an array index into the location of the glyph within a file and mostly arbitrary).  Font files contain a very fast N-1 mapping from `charcode` to `gind`, and FreeType has an API for this.  However, the reverse mapping is not directly available.

Prior to this change, Python dictionaries were created for the forward and reverse mapping, consuming a lot of memory for large fonts and being entirely redundant in the case of `ccode` to `gind`.

The `latex_to_bakoma` mapping *used to* map from LaTeX name to `gind` (for reasons that are lost in the sands of time, and are different from every other table in this file).  Since there's now no `gind` to `ccode` mapping, there was no longer a way to get both `gind` and `ccode` from the LaTeX name.  This table has now been changed to map LaTeX names to `ccode` instead (and this was done automatically by a script, so I'm reasonably confident everything is correct).

Sorry for the long explanation.  It's all bizarro.